internal/operators/ocs/validations.go

```diff
@@ -128,40 +123,56 @@ func (o *ocsValidator) ValidateRequirements(cluster *models.Cluster) (api.Valida
 	return api.Failure, status
 }
 
-func (o *ocsValidator) nodeResources(host *models.Host, cpuCount *int64, totalRAM *int64, diskCount *int64, hostsWithDisks *int64, insufficientHosts *[]string) (bool, error) {
+func (o *operator) checkInventoryStatus(r resources) (api.ValidationStatus, error) {
```

Comment:
This function is not required - its logic can be combined with that of the other node resource checks